JavaScript/Popup.js

```diff
@@ -0,0 +1,189 @@
+    'featured image': './images/tonic-img-1.png',
+    'featured image desk': './images/facebook-360-desktop.svg',
+    technologies: ['html', 'css', 'javascript'],
+    'link to live version':
+            '#/',
+    'link to source':
+            '#',
+    id: 2,
+  },
+  {
+    name: 'Multi-Post Stories',
+    title: 'Uber Navigation',
+    description:
+            'Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry\'s standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essent',
+    'featured image': './images/tonic-img-1.png',
+    'featured image desk': './images/uber-nav-desktop.svg',
+    technologies: ['html', 'css', 'javascript'],
+    'link to live version':
+            '#/',
+    'link to source':
+            '#',
+    id: 3,
+  },
+];
+ProjectInformation.forEach((element, i) => {
+  const work = document.querySelector('.card');
+  const card = document.createElement('article');
+  card.classList = 'card-section';
+  if (i === 1) {
+    card.setAttribute('id', 'reverce');
+  } else if (i === 3) {
+    card.setAttribute('id', 'reverce');
+  }
+
+  const popupcard = `
+    <img src="${ProjectInformation[i]['featured image']}" alt="Snapshot Portfolio" class="tonic-img-mobile">
+  <img src="${ProjectInformation[i]['featured image desk']}" alt="Snapshot Portfolio"
+      class="tonic-img-desktop">
+  <article class="card-section-text">
+  <h2 class="project-title-heading text-mobile">${ProjectInformation[i].name}</h2>
+  <h2 class="project-title-heading text-desktop">${ProjectInformation[i].title}</h2>
+  
+      <article class="card-sub-container">
+          <h2 class="title-canopy text-mobile">CANOPY</h2>
+          <h2 class="title-canopy text-desktop">CANOPY</h2>
+          <img src="./images/Counter.png" alt="Counter">
+          <h2 class="title-developer text-mobile">Back End Dev</h2>
+          <h2 class="title-developer text-desktop">Back End Dev</h2>
+          <img src="./images/Counter.png" alt="Counter">
+          <h2 class="title-year text-mobile">2015</h2>
+          <h2 class="title-year text-desktop">2015</h2>
+      </article>
+      <p class="desciption-p-mobile text-mobile">
+          A daily selection of privately personalized reads; no accounts or sign-ups required.
+      </p>
+      <p class="desciption-p-mobile text-desktop">
+          A daily selection of privately personalized reads; no accounts or sign-ups required.
+      </p>
+      <ul class="pro-language">
+      <li>
```

Comment:
**_To match the Figma design kindly fix these issues for the desktop version of the popup :_**

- [ ] kindly add the missing section below the main title
- [ ] the popup should be centered on the page
- [ ] kindly add the blurred background behind the popup
- [ ] the main image should take almost the fullwidth with some padding 
- [ ] kindly add space between the action buttons See Live & See Source you can use flex and justify-around/between, and please add the button interaction styling such as Hover, press, etc ...


**_figma:_**
![image](https://user-images.githubusercontent.com/89282221/216045504-d9cb510b-d312-4290-9c86-d3b0e3099123.png)